options/posix/generic/unistd-stubs.cpp

```diff
@@ -487,7 +487,12 @@ long pathconf(const char *, int name) {
 }
 
 int pause(void) {
-	__ensure(!"Not implemented");
+	MLIBC_CHECK_OR_ENOSYS(mlibc::sys_pause, -1);
+	if(int e = mlibc::sys_pause(); e) {
```

Comment:
Is it possible for `sc_error(e) != e` in case of error? If so, this return value is wrong